### DISCUSSION TITLE
How can I add/remove items to the "Programs" tab of a Bottle?

### DISCUSSION THREAD

**Discussion by felipehw** [bottlesdevs/Bottles#139] [Q&A] [answered]
Hi again :D

How can I add a Windows standalone executable to the "Programs" tab of a Bottle?
How can I remove an item from "Programs" tab of a Bottle? (It added the uninstaller of a program but not the program itself)

![Screenshot from 2021-02-26 22-16-27](https://user-images.githubusercontent.com/4523897/109370732-5dd02080-7880-11eb-9d8b-00e4d0965cd1.png)

**Comment by mirkobrombin** [comment, marked as answer]
At this time is not possible to add programs manually. Also you need to use the wine uninstaller to remove programs.

**Reply by felipehw** [reply]
Thanks!
